app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        return round(sum(self.wash_single_car(car) for car in cars), 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * (self.average_rating / self.distance_from_city_center),
+            1
+        )
+
+    def wash_single_car(self, car: Car) -> float:
+        if car.clean_mark >= self.clean_power:
+            return 0
+
+        price = self.calculate_washing_price(car)
+        car.clean_mark = self.clean_power
+
+        return price
+
+    def rate_service(self, rate: float) -> None:
+        self.average_rating = round(
+            ((self.average_rating * self.count_of_ratings) + rate)
+            / (self.count_of_ratings + 1),
+            1
+        )
+        self.count_of_ratings += 1
```

Comment:
In order not to perform the same operation (+1) twice, you can move this addition before calculating the average rating.